tests/utils/verification/volumeproperties.go

```diff
@@ -29,10 +29,6 @@ import (
 	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
 )
 
-const (
```

Comment:
I removed this because there's a conflict on this constant name after splitting this file into two. Also we need to refactor this part anyway (I've already added TODO on both functions). We should have a generic poller function, which accepts a maximum time limit instead of the maximum attempts as the input.